pythonforandroid/recipe.py

```diff
@@ -156,10 +156,10 @@ def report_hook(index, blksize, size):
             while True:
                 try:
                     urlretrieve(url, target, report_hook)
-                except OSError as e:
+                except OSError:
                     attempts += 1
                     if attempts >= 5:
-                        raise e
+                        raise
```

Comment:
Minor changes that I came across, doesn't harm, but sightly simplify things.
More serious refactoring will come after this module is properly covered by unit tests